apps/emqx_ldap/src/emqx_ldap_bind_worker.erl

```diff
@@ -58,14 +58,12 @@ on_stop(InstId, _State) ->
 
 on_query(
     InstId,
-    {bind, Data},
+    {bind, DN, Data},
     #{
-        base_tokens := DNTks,
         bind_password := PWTks,
         bind_pool_name := PoolName
     } = State
 ) ->
-    DN = emqx_placeholder:proc_tmpl(DNTks, Data),
```

Comment:
I think we can keep this. 
If DN is not a template, rendering it should be just a no-op

Reply:
should probably deprecate `base_dn` field, and make it optional?